src-ext/TypeScript/ggvega/src/itmLayerArrayByCoord.ts

```diff
@@ -70,5 +70,13 @@ function itmLayerArrayByCoordCartesian(itmLayerArray: ItmLayer[], gsCoord: GG.Co
  */
 function itmLayerArrayByCoordFlip(itmLayerArray: ItmLayer[], gsCoord: GG.Coord): ItmLayer[] {
   // will do something
+
+  // exchange encoding.x and encoding.y
```

Comment:
This has to be more robust than looking at just `x` and `y`. It needs to take into account other aesthetics like `xmin`, `ymin`.

Reply:
Sorry, I have no idea. Can you give me a specific example?

As my understanding, the `Vl.Encoding` only has `x` and `y`. And `xmin`, `ymin` is part of them(for now, they could be part of the `scale`. Maybe there are other situations). So exchange `x` and `y` is enough?